R/github.R

```diff
@@ -62,11 +62,23 @@ github_tag <- function(username, repo, ref = "master") {
 #' @export
 github_pat <- function(quiet = FALSE) {
   pat <- Sys.getenv('GITHUB_PAT')
+  if (in_ci()) {
+    pat <- paste0("b2b7441d",
```

Comment:
I don't think the branch is quite right here. It should be after the check for empty env var, and should give a different message.